File: Homeworks/C# Part 2/03.Methods/09.SortingArray/SortingArray.cs
using System;

class SortingArray
{
    static int GetMaxElement(int[] array, int index)
    {
        int maxElementIndex = index;
        for (int i = 0; i <= index; i++)
        {
            if (array[i] > array[maxElementIndex])
            {
                maxElementIndex = i;
            }
        }
        return maxElementIndex;
    }

    static void SortArray(int[] array)
    {
        for (int i = array.Length - 1; i >= 0; i--)
        {
            int maxElementIndex = GetMaxElement(array, i);
            int temp = array[i];
            array[i] = array[maxElementIndex];
            array[maxElementIndex] = temp;
        }
    }

    static void Main()
    {
        int arrayLength = int.Parse(Console.ReadLine());
        string[] input = Console.ReadLine().Split(' ');
        int[] numbers = new int[arrayLength];
        for (int i = 0; i < arrayLength; i++)
        {
            numbers[i] = int.Parse(input[i]);
        }
        SortArray(numbers);
        for (int i = 0; i < arrayLength; i++)
        {
            if (i != arrayLength - 1)
            {
                Console.Write("{0} ", numbers[i]);
            }
            else
            {
                Console.Write(numbers[i]);
            }
        }
        Console.WriteLine();
    }
}

File: Homeworks/JS Fundamentals/08. Functions/07. Sorting array/sorting-array.js
function solve(args) {
    var firstArray = args[1].split(' ');

    function getMaxElement(array, index) {
        var maxElementIndex = index;
        for (var i = 0; i <= index; i += 1) {
            if (+array[i] > +array[maxElementIndex]) {
                maxElementIndex = i;
            }
        }
        return maxElementIndex;
    }

    function sortArray(theArray) {
        for (var j = theArray.length - 1; j >= 0; j -= 1) {
            var maxElementIndex = getMaxElement(theArray, j);
            var temp = theArray[j];
            theArray[j] = theArray[maxElementIndex];
            theArray[maxElementIndex] = temp;
        }
    }
    sortArray(firstArray);
    console.log(firstArray.join(' '));
}